python/cudf/cudf/core/column/decimal.py

```diff
@@ -0,0 +1,69 @@
+import more_itertools
```

Comment:
CI is failing due to the use of `more_itertools` here, which isn't part of the CI container. We are going to replace code that uses `more_itertools` with custom numba kernels instead (something we originally wanted to do in a follow-up).